workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RenderWorkflow.kt

```diff
@@ -186,11 +187,42 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
       // we don't surprise anyone with an unexpected rendering pass. Show's over, go home.
       if (!isActive) return@launch
 
+      var drainingActionResult = actionResult
+      if (runtimeConfig.contains(DRAIN_EXCLUSIVE_ACTIONS)) {
+        drain@ while (isActive &&
```

Comment:
Do we still need this `isActive` check since `yield` should throw if the job is cancelled?

Reply:
No, its technically not needed. I like it for explicitness sake /readability though. is there an impact of this that i'm not considering and we should remove it?